scripts/local/import_local_db_from_json.sh

```
@@ -0,0 +1,53 @@
+  else
+    echo "⚠️  File not found: $FILE"
+  fi
+done
+
+for collection in "${collections[@]}"
+do
+  IMPORT_FILE="/import/${collection}.json"
+  
+  # Check if the file contains only an empty array
+  CONTENT=$(docker exec "$CONTAINER" cat "$IMPORT_FILE")
+  if [ "$CONTENT" == "[]" -o "$CONTENT" == "" ]; then
```

Shellcheck (0.10.0)

[warning] Prefer [ p ] || [ q ] as [ p -o q ] is not well defined.

(SC2166)